scripts/release-notes.py

```diff
@@ -43,6 +43,17 @@ def num_commits_since_prev_tag(token, base_url):
     print(f"There are {num_commits} new commits since {prev_release_tag}")
     return num_commits
 
+categories = {
+        'CI': 'changelog:ci',
+        'Feature': 'changelog:feature',
+    }
```

Comment:
We can modify this object as we want the result to be. For example:

```
categories = {
        '## Changes related to CI': 'changelog:ci',
        '## New Features': 'changelog:feature',
    }
```
this would categorise the PRs and would make sure the other person can just copy-paste the console logs

Reply:
yes, we need to do that, and keep the order as in the current CHLG